Documentation/event-reference.md

```diff
@@ -0,0 +1,208 @@
+# Edge Network Extension Event Reference<!-- omit in toc -->
+
+## Table of Contents<!-- omit in toc -->
+- [Events handled by Edge](#events-handled-by-edge)
+  - [Edge request content](#edge-request-content)
+  - [Edge request identity](#edge-request-identity)
+  - [Edge update consent](#edge-update-consent)
+  - [Edge update identity (event processing)](#edge-update-identity-event-processing)
+  - [Edge consent response content](#edge-consent-response-content)
+  - [Edge identity reset complete](#edge-identity-reset-complete)
+- [Events dispatched by Edge](#events-dispatched-by-edge)
+  - [Edge identity response](#edge-identity-response)
+  - [Edge content response error](#edge-content-response-error)
+  - [Edge event response](#edge-event-response)
+
+## Events handled by Edge
+
+The following events are handled by the Edge extension client-side.
+
+### Edge request content
+
+This event is a request to process and deliver an Experience event to Edge Network. This event is created when the `Edge.sendEvent(ExperienceEvent)` API is called. This event is captured by the Edge Network extension's event listener in the Event Hub for processing and sent to the Edge Network.
+
+If the required `xdm` key is not present in the event data payload, the event is not sent to Edge Network.
+
+#### Event Details<!-- omit in toc -->
+
+| Event type                 | Event source                         |
+| -------------------------- | ------------------------------------ |
```

Comment:
We can remove this table since the next table L32 has the same info with the API.